SYS_ATL/new_analysis_core.py

```diff
@@ -377,6 +381,112 @@ def pop():
         assert False, "bad case"
 
 
+def aeNegPos(e,pos,env=None,res=None):
```

Comment:
The `push()`, `pop()`, `env` and `res` stuff is a lightweight version of a compiler pass FWIW.  I'm removing `push()` and `pop()` because they are dead code.

I'm sorry this code is hard to follow.  It's trying to implement a reverse dataflow in order to determine whether or not particular code positions are positive or negative.  This is essential for the bug fix we've talked about.